days/day-03/solutions/day03.zig

```diff
@@ -0,0 +1,66 @@
+const std = @import("std");
+const mem = std.mem;
+const io = std.io;
+const ArrayList = std.ArrayList;
+
+const MAX_FILE_SIZE = 20_000;
+
+const Map = struct {
+    bytes: []u8,
+    line_width: usize,
+    map_height: usize,
+
+    pub inline fn init(bytes: []u8) Map {
+        var line_width: usize = 0;
+        while (line_width < bytes.len) : (line_width += 1) {
+            switch(bytes[line_width]) {
+                '\n', '\r' => break,
+                else => continue
+            }
+        }
+
+        line_width += 1;
+
+        return Map {
+            .bytes = bytes,
+            .line_width = line_width,  // TODO: account for arbitrary line ending format
```

Comment:
Behold, the most boring collection of words in the universe: "arbitrary line ending format". :laughing: Do not mention this on a date :100:

Reply:
Can't promise anything 😁